core/shared/src/main/scala/fs2/Stream.scala

```diff
@@ -5361,19 +5371,8 @@ object Stream extends StreamLowPriority {
 }
 
 private[fs2] trait StreamLowPriority {
-  implicit def monadInstance[F[_]]: Monad[Stream[F, *]] =
-    new Monad[Stream[F, *]] {
-      override def pure[A](x: A): Stream[F, A] = Stream.emit(x)
-
+  implicit def monadInstance[F[_]]: StreamMonad[F] =
+    new StreamMonad[F] {
```

Comment:
Since `StreamMonad` is not an abstract class, you can omit the body (the `{` / `}` block) here.

Reply:
Thanks! :)
If I do this, I will move the `unit` method to the `StreamMonad`. And `monadErrorInstance` will also use that method.
But it don't use the override unit method before.